src/rpc.rs

```diff
@@ -19,7 +24,13 @@ pub struct JsonRpcService {
 }
 
 impl JsonRpcService {
-    pub fn new(bank: Arc<Bank>, rpc_addr: SocketAddr, exit: Arc<AtomicBool>) -> Self {
+    pub fn new(
+        bank: Arc<Bank>,
```

Comment:
clippy didn't tell you to make this `&Arc<Bank>`?  Adding the reference lets you get ride of the clone() from anyone that calls `new()` and instead there's just be one `clone()` in this function when you pass the `Arc` into the thread.

Reply:
✔️